src/main/java/Duke.java

```diff
@@ -19,10 +19,16 @@ public static void main(String[] args) {
                 break;
             } else if (next.equals("list")) {
                 for (int i = 0; i < counter; i++) {
-                System.out.printf("%d. %s\n",i +1,tasks[i]);
+                System.out.printf("%d.[%s] %s\n",i +1,tasks[i].getStatusIcon(),tasks[i].getDescription());
```

Comment:
I think it would be a good idea to extract a method from this. For example something like listTasks()